Frontend/src/pages/registration_administration/components/RegistrationActions.jsx

```diff
@@ -87,20 +103,45 @@ export default function RegistrationActions({ selected, refresh }) {
           <Button
             onClick={() =>
               changeStatus(
-                [...selected.accepted, ...selected.cancelled],
+                [
+                  ...selected.accepted,
+                  ...selected.cancelled,
+                  ...selected.waiting,
+                ],
                 'pending'
               )
             }
           >
             <UiIcon name="times" /> Reject
```

Comment:
"Reject" also seems unclear to me. I could image it sending them to any of the non-approved lists...